python/py-macresources/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=Portfile:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-macresources
+version             1.2
+
+categories          devel
+license             MIT
+platforms           darwin
+supported_archs     noarch
+maintainers         nomaintainer
+
+description         a library for working with legacy Macintosh resource forks
+long_description    A Python library and command line tools to work with Classic MacOS \
+                    resource forks on a modern machine.
+
+homepage            https://github.com/elliotnunn/macresources
+
+python.versions     36 37 38
```

Comment:
There is no need to add the older python versions for a new python module. Just use 38.

Reply:
ok, same on both PR's then I guess?